### ISSUE TITLE
An error in calculating MAPE for QoR tasks

### ISSUE THREAD

**Issue by Jedges** [NYU-MLDA/OpenABC#14]
/models/qor/SynthNetV1/utils.py [line 258](https://github.com/NYU-MLDA/OpenABC/blob/936a3b658a7c521390ee07dc5698a57c05e5be6b/models/qor/SynthNetV1/utils.py#L258C1-L258C108)，When calling mean_absolute_percentage_error for calculation, are the two parameters placed in reverse order? The first one in the function description is y_true, and the second one is y_pred, but the code uses y_pre as the first parameter. 
![image](https://github.com/user-attachments/assets/4f10bbac-5bd8-4810-b664-3beffbdc06f6)
I am not sure if my understanding is wrong, please let me know if I am wrong

**Comment by animeshbchowdhury** [comment]
Indeed, you pointed out an error. I did calculate the MAPE score incorrectly. It should have been reversed.

Can you make a PR with the change?

**Comment by Jedges** [comment]
Yes, I maked a PR with the change. But  I'm not sure if I'm doing the PR operation correctly, this is my first time using it.
